google/ggrc-core
# GGRC-5891 ACL roles and LCA are saved after delay on assessment page

# Issue description

There is a small delay (around a second) before a spinner appears. During this period there are no DOM changes and no HTTP requests, just JS processing. We are unsure how wait for this period to pass so we had to put a sleep into Selenium tests for this.
LCA are saved after delay.

# Steps to test the changes

1. Open assessment.
2. Edit ACL roles, or LCA.
3. Expected: Expected: Saving ACL or LCA called after minimal delay (4ms)

# Solution description

Used 'execute' instead for 'push' for deferredSave. It sets minimal delay before saving. We cant avoid this completely for correctly saving assessment after editing LCA or ACL roles. 

# Sanity checklist




# PR Review checklist
